Makefile

```diff
@@ -188,10 +188,6 @@ deploy-mutation: patch-image
 	  k8s.gcr.io/kustomize/kustomize:v${KUSTOMIZE_VERSION} build \
 	  --load_restrictor LoadRestrictionsNone \
 	  /config/overlays/dev_mutation | kubectl apply -f -
-	docker run -v $(shell pwd)/config:/config -v $(shell pwd)/vendor:/vendor \
```

Comment:
Are these changes meant to be in this PR?

Reply:
Yep, the pipeline made it harder than it needed to be to change the mutation manifest.

I could have broken it out, but that'd be another PR that'd need review/submission.